.circleci/config.yml

```diff
@@ -598,6 +602,12 @@ jobs:
           paths:
             - solana-programs/claimable-tokens/program/target
           key: claimable-token-deps-{{ checksum "solana-programs/claimable-tokens/program/Cargo.toml" }}
+      - run:
+          name: anchor tests
+          command: |
+            cd solana-programs/anchor/audius-data
+            npm i
```

Comment:
change this to yarn per anchor documentation - this doesn't install everything properly https://project-serum.github.io/anchor/getting-started/installation.html#install-solana

Reply:
Oops, thanks, fixing now

Reply:
yeah let's follow basic installation steps moving forward